docker/sirius-lpa-frontend/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:20.3.0-alpine3.17 as asset-env
+FROM node:20.2.0-alpine3.17 as asset-env
```

Comment:
Why is this downgraded?

Reply:
I was having trouble running 20.3 locally because of issues on my machine, so just downgraded to quickly unblock myself